packages/sparqlee/README.md

```diff
@@ -0,0 +1,364 @@
+# Sparqlee
```

Comment:
Could we align this README to the other README's in comunica? For example: https://github.com/comunica/comunica/tree/master/packages/data-factory

Reply:
The links on the classes seem to be broken. What should the point to?

Reply:
There's a search function on https://comunica.github.io/comunica/

E.g. https://comunica.github.io/comunica/classes/_comunica_data_factory.BlankNodeScoped.html